backend/server.js

```diff
@@ -15,15 +52,93 @@ const port = process.env.PORT || 8080
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
+
+const authenticateUser = async (req, res, next) => {
+  try {
+    const user = await User.findOne({ accessToken: req.header('Authorization') })
+    if (user) {
+      req.user = user
+      next()
+    } else {
+      res.status(401).json({ loggedIn: false })
+    }
+  } catch (err) {
+    res.status(403).json({ message: 'Access token missing or invalid', errors: err.errors })
+  }
+}
+
 app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Tiago was here!and Ivett too!!')
+})
+// Signup post
+app.post('/users', async (req, res) => {
+  console.log('ivett')
```

Comment:
Sometimes I use the test string 'asfd' or 'here' or `Processing '/users' : req.body}` 😉